# Aufgabe 2

a) CO_2-Menge

In [1]:
n = 6 # Personen
dV_co2_person = 23e-3 # 23 l/h in m**3/h
dV_co2 = n*dV_co2_person
dV_co2

0.138

Es werden 138 Liter CO_2 je Stunden produziert.

In [2]:
Delta_t = 12 # Stunden
V_co2 = dV_co2*Delta_t # m**3
V_co2

1.6560000000000001

In 12 Stunden ergibt das 1.656 Kubikmeter CO_2.

b) Erforderlicher Außenluftvolumenstrom

$$
  \dot V_{au} = \dfrac{\dot V_{co2}}{k_{zul}-k_{au}}
$$

In [3]:
k_zul = 1000e-6 # 1000 ppM
k_au = 400e-6 # 400 ppM

dV_au = dV_co2/(k_zul-k_au)
dV_au # m**3/h

230.0

Um das CO_2 abzutransportieren, werden 230 Kubikmeter pro Stunde Außenluft benötigt.

c) Luftwechselzahl des Raumes

In [4]:
V_ra = 150*2.5 # m**3
V_ra

375.0

In [5]:
beta = dV_au/V_ra
beta

0.6133333333333333

Der zugehörige Luftwechsel ist $0.613\,\mathrm{h}^{-1}$.

d) Elternschlaftzimmer

In der Nacht wird von einem reduzierten CO_2-Ausstoß der Eltern ausgegangen, da diese schlafen.

In [7]:
V_eltern = 15*2.5 # m**3

dV_co2_eltern = 2*15e-3 # Reduzierter CO_2-Ausstoß der Eltern

V_co2_eltern = 8*dV_co2_eltern
V_co2_eltern # m**3

0.24

Die Eltern produzieren 240 Liter CO_2 in der Nacht.

In [8]:
k_eltern_ungelueftet = k_au + V_co2_eltern/V_eltern
k_eltern_ungelueftet*1e6 # in ppM

6800.0

Das führt auf den völlig inakzeptablen Wert von 6800 ppM CO_2 im Elternschlafzimmer.

e) Erforderlicher Zuluftvolumenstrom im Elternschlafzimmer

In [9]:
dV_au_eltern = dV_co2_eltern/(k_zul - k_au)
dV_au_eltern

49.99999999999999

Um den Raum zu belüften, werden 50 Kubikmeter Außenluft pro Stunde benötigt.